### PR TITLE
[WU] Suppress hardcoded unit in channel description and documentation

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -344,26 +344,26 @@
 
 	<channel-type id="windDirection-degrees" advanced="true">
 		<item-type>Number:Angle</item-type>
-		<label>Wind Direction (°)</label>
-		<description>Wind direction in degrees</description>
+		<label>Wind Direction (angle)</label>
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml]: I would suggest to remove "(angle)" from the label - we do not have that on other labels either and the state will be shown next to it with a unit anyhow.

File: extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -344,26 +344,26 @@
...
 		<category>Wind</category>
 		<state readOnly="true" min="0" max="360" pattern="%.0f %unit%">
 		</state>
 	</channel-type>
 
 	<channel-type id="maxWindDirection-degrees" advanced="true">
 		<item-type>Number:Angle</item-type>
-		<label>Maximum Wind Direction (°)</label>
-		<description>Maximum wind direction in degrees</description>
+		<label>Maximum Wind Direction (angle)</label>
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml]: dito

**Comment by htreu** [comment]
@lolodomo, please create another PR with Kai's comments addressed. Thanks.

File: extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml (hunk 1 above)
Comment (lolodomo) [reply extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml]: This is only to distinguish the 2 direction channels, one with an angle (90 for example) and the other with a text (NE for example).

Comment (kaikreuzer) [reply extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml]: I'd think that it is enough to show this distinction in the description - as the label is used in the UIs in the end (where this info is rather disturbing).

Comment (lolodomo) [reply extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml]: Then the user would see 2 channels with exactly the same label in Paper UI. Very confusing. And as getting the channel description in Paper UI is all except intuitive...

Comment (kaikreuzer) [reply extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml]: Hm, you are right - I thought that the description is shown on the Thing detail page, but it isn't (only hidden behind a long-press).
So ok, let's keep that distinction in the label then...

**Pull request merged**